### ISSUE TITLE
OpenSSL 1.0.x doesn't load on RHEL 7

### ISSUE THREAD

**Issue by pshipton** [ibmruntimes/openj9-openjdk-jdk#122]
On RHEL 7, the libcrypto library is not loaded. Looking at the library names on the machine, the library used for 1.0.x is not currently supported.

```
        libcrypto.so.10 (libc6,x86-64) => /lib64/libcrypto.so.10
        libcrypto.so.10 (libc6) => /lib/libcrypto.so.10
        libcrypto.so.6 (libc6,x86-64) => /lib64/libcrypto.so.6

-rwxr-xr-x 1 root root 1440264 Mar  4  2016 libcrypto.so.0.9.8e
-rwxr-xr-x 1 root root 2512264 May 17  2017 libcrypto.so.1.0.2k
lrwxrwxrwx 1 root root      19 Sep  3  2017 libcrypto.so.10 -> libcrypto.so.1.0.2k
lrwxrwxrwx 1 root root      19 Mar 20  2016 libcrypto.so.6 -> libcrypto.so.0.9.8e
```
```
    const char *libname = "libcrypto.so.1.1";    // Library name for OpenSSL 1.1.0 and 1.1.1
    const char *oldname = "libcrypto.so.1.0.0";  // Library name for OpenSSL 1.0.2
    const char *symlink = "libcrypto.so";        // Library name for possible symbolic links
```

**Comment by pshipton** [comment]
@groeges

**Comment by andrewcraik** [comment]
Just linking https://github.com/eclipse/openj9/issues/6114 since it might be something that could be done as a side-effect to make it easy to verify the changes work.

**Comment by pshipton** [comment]
@andrew-m-leonard @groeges any outlook for getting this fixed? It would be nice to have the fix in the OpenJ9 0.15 release.

**Comment by pshipton** [comment]
@groeges @lumpfish I see changes being merged to fix this. Please also put the fix into the 0.15 branches.

**Comment by groeges** [comment]
Issue should be resolved on jdkNext, jdk13, jdk12, jdk11 and jdk8.
PRs created and merged for all the above versions.
When Andrew picks up the quarterly updates next week these additional changes will be merged into the 0.15 branches.
